go-controller/pkg/libovsdbops/acl.go

```diff
@@ -148,7 +148,7 @@ func DeleteACLs(nbClient libovsdbclient.Client, acls ...*nbdb.ACL) error {
 			Model:          acl,
 			ModelPredicate: func(item *nbdb.ACL) bool { return isEquivalentACL(item, acl) },
 			ErrNotFound:    false,
-			BulkOp:         false,
+			BulkOp:         true,
```

Comment:
@flavio-fernandes : PTAL. this was the spot I was talking about, if users ended up with multiple ACLs on the same match, we fail hard and don't delete them while I think for deletion we should be fine. We should probably re-evaluate the history of `IsEquivalentACL` in another PR..

Reply:
yes, @tssurya ! I think this is needed, since we now expect to find match more than 1 row.
/lgtm